content/tools/extensions/AzureDevOpsExtension.md

```diff
@@ -0,0 +1,63 @@
+<!--
+title: "Contrast Integration for Azure DevOps" 
+description: "Extension to integrate Contrast in your deployment workflow"
+tags: "tools azure devops integration extension deployment"
+-->
+
+
+Integrate Contrast in your deployment workflow with the Azure DevOps extension. The following instructions guide you through the steps to set up and configure the extension for your Contrast instance. 
+
+## Before You Start 
+
+Before you begin to set up the extension, make sure that you have the privileges to install a Microsoft extension. If you lack the necessary privileges, you can [request the extension](https://docs.microsoft.com/en-us/azure/devops/marketplace/request-extensions?view=azure-devops-2019) for a project.
+
+## Setup and Configuration
+
+### Step one
+
+* Follow [Microsoft's instructions](https://docs.microsoft.com/en-us/azure/devops/marketplace/install-extension?view=azure-devops-2019) to install the Contrast extension, **Contrast Integration**. 
+
+### Step two
+
+* Go to your **Project settings** at the bottom of you side bar.
+
+* In the **Pipelines** section of the settings menu, select **Service connections**.
+
+ <a href="assets/images/AzureDevOps_connection_settings.png" rel="lightbox" title="Service Connection Settings"><img class="thumbnail" src="assets/images/AzureDevOps_service_connection_settings.png"/></a>
+
+### Step three
+
+* Click over the **+ New Service connection** button (shown in the image in the previous step), and select **Contrast Server Connection**.
+* Complete all the fields with the required data. You can find the values for all the fields in the Contrast UI by going to the [**user menu > Your Account > Profile** page](user-account.html#profile).
+
+<a href="assets/images/AzureDevOps_service_connection.png" rel="lightbox" title="Service Connection fields"><img class="thumbnail" src="assets/images/AzureDevOps_service_connection.png"/></a>
+
+> **Note: ** Your **Contrast URL** should not include */Contrast* at the end; only the host is required.
+
+## Configuration for Release Gate
+
+### Step one
+
```

Comment:
@rduran0 Are there any images for this step?

Reply:
No. But I can get you some. And now that you mention it, that should make it less confusing for newcomers.